app/assets/stylesheets/application.scss

```diff
@@ -126,4 +126,45 @@ h6 {
 
 .nav-feature {
   color: black;
+
+}
+
+.center-img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.item-show-img {
+  width: auto;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+
+}
+
+.item-background-img {
+
+}
+
+#main {
+  position: relative;
+  text-align: center;
+  h1 {
+    color: white;
+  }
+
+}
+
+#main:after {
+   content : "";
+   display: block;
+   position: absolute;
+   top: 0;
+   left: 0;
+   background-image: url("/assets/plate.jpg");
+   width: 100%;
+   height: 100%;
+   opacity : 0.4;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Property name should be immediately followed by a colon